src/components/ApplicantList.jsx

```diff
@@ -0,0 +1,101 @@
+function ApplicantList() {

```

Comment:
This component should receive an array of applicants as a prop, and render accordingly.

Reply:
done